src/brogue/Items.c

```diff
@@ -5209,7 +5221,7 @@ short hiliteTrajectory(const pos coordinateList[DCOLS], short numCells, boolean
         } else if (!passThroughMonsters && pmap[x][y].flags & (HAS_MONSTER)
                    && (playerCanSee(x, y) || player.status[STATUS_TELEPATHIC])) {
             monst = monsterAtLoc((pos){ x, y });
-            if (!(monst->bookkeepingFlags & MB_SUBMERGED)
+            if ((!(monst->bookkeepingFlags & MB_SUBMERGED) ||  playerCanTargetSubmergedMonster(monst)) //todo?
```

Comment:
what's todo here?

Reply:
I marked references to MB_SUBMERGED for investigation. Forgot to remove that one. I'll remove it.

Reply:
Ok done.